apps/synapse-portal-framework/src/components/elportal/ELBrowseToolsPage.tsx

```diff
@@ -0,0 +1,293 @@
+import { SynapseComponents, FeaturedToolsList } from 'synapse-react-client'
+import Layout from '../Layout'
+import React from 'react'
+import { Box, Button, Link, TextField, Typography } from '@mui/material'
+import { Query, TextMatchesQueryFilter } from '@sage-bionetworks/synapse-types'
+// import PopularSearches from '../PopularSearches'
+import pluralize from 'pluralize'
+import Ecosystem from '../csbc-home-page/Ecosystem'
+
+type Category = {
+  resourceName: string
+}
+
+const categories: Category[] = [
+  { resourceName: 'Cell Lines' },
+  { resourceName: 'Gene Targets' },
+  { resourceName: 'Biomarkers' },
+  { resourceName: 'Chemoinformatics' },
+]
+
+export type ELBrowseToolsPageProps = {
+  popularSearchesSql: string
+  toolsSql: string
+}
+
+const ELBrowseToolsPage = (props: ELBrowseToolsPageProps) => {
+  const { toolsSql } = props
+  // const { popularSearchesSql } = props
+  const [searchText, setSearchText] = React.useState<string>('')
+  const gotoExploreTools = () => {
+    window.location.assign('/Explore/Computational%20Tools')
+  }
+
+  const gotoExploreToolsWithSelectedResource = (selectedResource: string) => {
+    const query: Query = {
+      sql: toolsSql,
+      selectedFacets: [
+        {
+          concreteType:
+            'org.sagebionetworks.repo.model.table.FacetColumnValuesRequest',
+          columnName: 'resourceType',
+          facetValues: [selectedResource],
+        },
+      ],
+    }
+    window.location.assign(
+      `/Explore/Computational%20Tools?QueryWrapper0=${JSON.stringify(query)}`,
+    )
```

Comment:
For reference, instead of using `location.assign`, we should pull in `useHistory` (`useNavigate` in react-router v6) and use that to push to prevent a full reload.

I'll probably review all of the places where we use `location` in the work I'm currently doing, so no need to change this now.

Reply:
Need to URLEncode the stringified query?